### ISSUE TITLE
Accidentaly uploaded 2 JPGs instead of PSDs

### ISSUE THREAD

**Issue by PandaHugMonster** [avocode/avocode#498]
1. Would be good to add rastr-pictures (JPG) for the exact Design - to have ability to compare with the final design and the parsed PSD (Because PSD not always parsed correctly)
2. After accidental uploading of wrong files - I can't delete them before I finish saving - this is really bad idea.
3.

-----
App v.2.4.3-3630f5a
Source UID: 126548
```
Error: invalid stored block lengths
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by vojtatranta** [comment]
1. We are currently finalizing "preview design" feature, which does pretty much the same thing as you suggest.
2. We added this request as an enhancement to our TODO list - #499
3. That is a known bug we know about, we have it on the schedule.

Duplicate of #447

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
